src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java

```diff
@@ -335,6 +335,7 @@
     public static final String HEARING_NUMBER_MISMATCH_ERROR_MESSAGE = "The hearing number provided did not match with any of the hearing numbers contained within this case";
     public static final String DUPLICATE_JURISDICTION_CODE_ERROR_MESSAGE = "Duplicate Jurisdiction Code(s) : ";
     public static final String MISSING_JURISDICTION_OUTCOME_ERROR_MESSAGE = "A Jurisdiction outcome is required before the case can be closed";
+    public static final String MISSING_JURISDICTION_MESSAGE = "You cannot close this case as there is currently no jurisdiction and no jurisdiction outcome associated with it";
```

Comment:
Could this not just be 'A Jurisdiction is required before the case can be closed'?
Or with your message, no need to mention jurisdiction outcome as no jurisdictions exist anyway??

Reply:
I think Tom has suggested her this text in the slack channel. She is using that only.

Reply:
Yh Angela told me that after.